sdcm/cluster_gce.py

```diff
@@ -45,6 +45,8 @@ def __init__(self, gce_instance, gce_service, credentials,
                           cluster.TEST_DURATION)
             self._instance_wait_safe(self._gce_service.ex_set_node_tags,
                                      self._instance, ['keep-alive'])
+        self._instance_wait_safe(self._gce_service.ex_set_node_metadata,
+                                 self._instance, {'workspace': cluster.WORKSPACE, 'uname': ' | '.join(os.uname())})
```

Comment:
the same

Reply:
Replied.

Reply:
Tags in gce only support lowercase letters, numbers, and hyphens, so I added new values to metadata. Here we have to use two calls.

Reply:
In GCE we don't set tag `Name` as AWS, so nothing needs to be fixed here.